matrix-vis.js

```diff
@@ -105,11 +105,24 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
+        var Type = " ";
+        var Sender = " ";
+        var m = " ";
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            Type=JSON.stringify(nodes.get(params.nodes[0]).blob.type, undefined, 2);
+            Sender=JSON.stringify(nodes.get(params.nodes[0]).blob.sender, undefined, 2);
+            t1="Event Type : "+Type;
+            t2="Creator : "+Sender;
+            if (Type==JSON.stringify("m.room.member")) {
+            m=JSON.stringify(nodes.get(params.nodes[0]).blob.content.membership, undefined, 2);
```

Comment:
You constantly do `nodes.get(params.nodes[0]).blob` - it may be worth you pulling this out to a separate variable so you don't have to keep typing it, e.g. `var event = nodes.get(params.nodes[0]).blob;`